src/ethereum/api/collection.ts

```diff
@@ -132,6 +135,13 @@ const getRaritiesQuery = () => gql`
 export const COLLECTIONS_URL = env.get('COLLECTIONS_GRAPH_URL', '')
 
 export class CollectionAPI extends BaseGraphAPI {
+  logger: ILoggerComponent.ILogger
+
+  constructor(url: string) {
+    super(url)
+    this.logger = createConsoleLogComponent().getLogger('GraphAPI')
```

Comment:
Maybe it's worth it to call it `CollectionsGraphAPI` ? or are we trying to group all graph api logs?

I say this because we also have third parties :)

Reply:
You're right! Renamed.